config/jobs/kubernetes-sigs/secrets-store-sync-controller/secrets-store-sync-controller-config.yaml

```diff
@@ -104,6 +104,174 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-sync-controller-image-scan
       description: "Run vulnerability scans for Secrets Store Sync controller images."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-sync-controller-e2e-provider-k8s-1-27-15
+    cluster: eks-prow-build-cluster
+    decorate: true
+    decoration_config:
+      timeout: 10m
+    always_run: true
+    optional: true
+    path_alias: sigs.k8s.io/secrets-store-sync-controller
+    branches:
+    - ^main$
+    labels:
+      # this is required because we want to run kind in docker
+      preset-dind-enabled: "true"
+      # this is required to make CNI installation to succeed for kind
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240705-131cd74733-master
+        command:
+          - runner.sh
+        args:
+          - bash
+          - -c
+          - >-
+            make docker-build local-setup-e2e run-e2e-provider-tests
```

Comment:
You can keep `docker-build` and  `local-setup-e2e` dependent target for `run-e2e-provider-tests`. We can keep this clean and just have one target here. This will also give us flexibility in terms of just making changes in Makefile rather than coming back here updating when we need to.